### PR TITLE
Removed the First Step and made it the first step of Part IV in the readme file

### PR DESCRIPTION
**Pull request by Bush1do**
This is mainly for clarification before editing the actual HTML

### CODE REVIEW

**Comment by MaxWofford** [comment]
Thanks for the PR @Bush1do! Will review it asap.

**Comment by MaxWofford** [comment]
There are a few more steps to fix here. Right now you need a GitHub repo to create the Cloud9 workspace, but don't create your GitHub account or repo until after you have a Cloud9 workspace. Ping me once you've fixed it.

**Comment by Bush1do** [comment]
So where does you paste the HTTPS link of a repo in c9?

File: workshops/personal_website/README.md [hunk 1]
```diff
@@ -313,11 +278,48 @@ Remember to save both `index.html` and `styles.css`! We'll once again use the te
...
+
+   ![](img/gh_create_new_repo.png)
+
+3. Yay! You've created your very first repository! Now, copy the "HTTPS link" at the top. You will need this link for the next step.
+
+   ![](img/gh_copy_link.gif)
+
+### 3) Launch!
+
+1. Back in Cloud9, under "Clone from Git or Mercurial URL", paste the HTTPS Link that you copied from your GitHub repository.
```
Comment (MaxWofford) [inline workshops/personal_website/README.md]: Where is the "Clone from Git or Mercurial URL" area in Cloud9?

Comment (Bush1do) [reply workshops/personal_website/README.md]: Copied that from the first step before c9 was setup

**Comment by MaxWofford** [comment]
@Bush1do: before they run `git add`, they'll need to run these commands to set up the repo:

```sh
git init
git remote add origin https://github.com/USERNAME/USERNAME.github.io
```

File: workshops/personal_website/README.md [hunk 2]
```diff
@@ -297,11 +297,11 @@ GitHub allows us to host our website using a service called GitHub Pages. This m
...
 Using the terminal allows us to do things on computers which you can't usually do with a standard graphical user interface. We don't expect you to be a terminal master by the end of this tutorial, but with this initial exposure hopefully you'll be able to continue learning more about it over time!
 
-5.  In the bottom left of your C9 dashboard is a section called “Add-on Services”. Click “activate” next to the “GitHub” icon (if you haven’t already).
-When the pop up appears, tell it to allow access to your account. This should complete the link between GitHub account and your C9 account.
-While still on the dashboard, look to the right under “Account Settings” for the link “Show your SSH key”. Click on it, and copy the value which appears.
-Go to the GitHub SSH settings page, and click “Add SSH Key”
-Enter the title “C9”, paste the SSH Key into the “Key” box, and click “Add Key”.
+5.  On the dashboard, click the gear icon on the right to access your Settings and look on the left for the link “SSH keys”. Click on it, and copy the value which appears under "Connect to your private git repository".
+
+6.  Go back to GitHub and your settings page, then click “SSH and GPG Key”
```
Comment (MaxWofford) [inline workshops/personal_website/README.md]: Why bother with using an SSH key? They should just use `git init` and `git remote add <remote> <url>` https://github.com/hackclub/hackclub/pull/1062#issuecomment-337032048

Comment (Bush1do) [reply workshops/personal_website/README.md]: Not familiar with git

Comment (MaxWofford) [reply workshops/personal_website/README.md]: That's fine, just make sure to ask questions when you don't know something clearly. Saves both of our time if we don't have to keep changing the PR.

**Comment by zachlatta** [comment]
Closing due to lack of activity.

**Pull request closed without merging**